terminology.md

```diff
@@ -151,45 +106,6 @@ The [CoverParenthesizedExpressionAndArrowParameterList](https://tc39.es/ecma262/
 
 The definition of [covering](https://tc39.es/ecma262/#sec-syntactic-grammar) in the ECMAScript specification, used to check whether it's valid to reinterpret one grammatical production as another.
 
-## Web compatibility/"Don't break the web"
-
-### Definition
-
-A change to JavaScript is considered "**web compatible**" if it preserves the current behavior of existing websites. If it changes the behavior of existing websites, it's considered to "**break the web**".
```

Comment:
a) i think this diff should be done in a separate PR, and b) i'm not sure alphabetically is the best approach

Reply:
if we're not sorting the glossary, there's no reason to merge any of this